src/main/webapp/app/shared/tooltip/PubMedArticleTooltip.tsx

```diff
@@ -8,7 +8,7 @@ import _ from 'lodash';
 import { formatDate } from 'app/shared/util/utils';
 
 const getPubMedContent = (pubMedDto: PubMedDTO) => {
-  if (pubMedDto.additionalInfo?.abstractTexts?.length > 1) {
+  if (pubMedDto.additionalInfo?.abstractTexts && pubMedDto.additionalInfo?.abstractTexts?.length > 1) {
```

Comment:
Could just use `pubMedDto.addtionalInfo?.abstractTexts?.length > 1`

Reply:
I get this error from typescript if I do that. `1. 'pubMedDto.additionalInfo.abstractTexts.length' is possibly 'undefined'. [18048]`